README.md

```diff
@@ -99,10 +106,13 @@ Installation steps:
 4. Make a copy of `db/schema.rb.example` and place it at `db/schema.rb`.
 5. Make a copy of `config/database.yml.sqlite.example` and place it at `config/database.yml`
 6. Run `rake db:setup` to set up the database
-7. Install static assets (like external javascript libraries, fonts) with `bower install` 
-8. Start rails with `passenger start` from the Rails root and open http://localhost:3000 in a web browser. 
-9. Wheeeee! You're up and running! Log in with test usernames "user", "moderator", or "admin", and password "password". 
-10. Run `rake test:all` to confirm that your install is working properly. For some setups, you may see warnings even if test pass; [see this issue](https://github.com/publiclab/plots2/issues/440) we're working to resolve. 
+7. Install static assets (like external javascript libraries, fonts) with `bower install`
+8. Install solr engine `rails generate sunspot_rails:install`
```

Comment:
Is this really how you install it? It looks like this is a step after installing?